example/rider/rider_v2.yaml

```diff
@@ -0,0 +1,73 @@
+static_resources:
+  listeners:
+  - name: main
+    address:
+      socket_address:
+        address: 0.0.0.0
+        port_value: 8080
+    filter_chains:
+    - filters:
+      - name: envoy.http_connection_manager
+        typed_config:
+          "@type": type.googleapis.com/envoy.extensions.filters.network.http_connection_manager.v3.HttpConnectionManager
+          stat_prefix: ingress_http
+          codec_type: auto
+          route_config:
+            name: local_route
+            virtual_hosts:
+            - name: local_service
+              domains:
+              - "*"
+              # Plugin config here applies to the VirtualHost
+              #
+              # typed_per_filter_config:
+              #   proxy.filters.http.rider:
+              #     "@type": type.googleapis.com/proxy.filters.http.rider.v3alpha1.RouteFilterConfig
+              #     plugins:
+              routes:
+              - match:
+                  prefix: "/"
+                route: 
+                  cluster: web_service
+                typed_per_filter_config:
+                  proxy.filters.http.rider:
+                    "@type": type.googleapis.com/proxy.filters.http.rider.v3alpha1.RouteFilterConfig
+                    plugins:
+                      - name: fun_test_v2
+
+          http_filters:
+          - name: proxy.filters.http.rider
+            typed_config:
+              "@type": type.googleapis.com/proxy.filters.http.rider.v3alpha1.FilterConfig
+              plugin:
+                vm_config:
+                  package_path: "/home/starrynight/workspace/rider/?/init.lua;/home/starrynight/workspace/rider/?.lua;"
+                code:
+                  local:
+                    filename: /home/starrynight/workspace/rider/plugins/test/fun_test_v2.lua
```

Comment:
Please remove personal info in this example yaml

Reply:
done